2-Authorization-I/2-call-arm/SPA/src/azureManagement.js

```diff
@@ -0,0 +1,90 @@
+import { msalInstance } from './index';
+
+import { SubscriptionClient } from '@azure/arm-resources-subscriptions';
+import { BlobServiceClient } from '@azure/storage-blob';
+import { protectedResources, storageInformation } from './authConfig';
+
+/**
+ * Returns a subscription client object with the provided token acquisition options
+ */
+export const getSubscriptionClient = async () => {
+    const browserCredential = new BrowserCredential(msalInstance, protectedResources.armTenants.scopes);
+    await browserCredential.prepare();
+
+    const client = new SubscriptionClient(browserCredential);
+    return client;
+};
+
+/**
+ * Returns a blob service client object with the provided token acquisition options
+ */
+export const getBlobServiceClient = async () => {
+    const browserCredential = new BrowserCredential(msalInstance, protectedResources.armBlobStorage.scopes);
+    await browserCredential.prepare();
+    const client = new BlobServiceClient(
+        `https://${storageInformation.accountName}.blob.core.windows.net`,
+        browserCredential
+    );
+    return client;
+};
+
+
+/**
+ * This class implements TokenCredential interface more information see:
+ * https://docs.microsoft.com/en-us/javascript/api/@azure/core-auth/tokencredential?view=azure-node-latest
+ */
+class BrowserCredential {
+    publicApp;
+    hasAuthenticated = false;
+    scopes;
```

Comment:
Nit:

Do we not need a `let` declaration before the variables here?

Reply:
It's an attribute not a variable can be defined with let